README.md

```diff
@@ -16,6 +16,7 @@ This is Frets on Fire X, a highly customizable rhythm game supporting many modes
 
 Setup
 -----
+[How-To-Guide For Linux Users] (/doc/Linux_Setup_Guide.md)
```

Comment:
A space between the text and the link is breaking the rendering.
About the file, it should be in `doc/source/quickstart/`.

Reply:
(not intentional at all). Good catch. Fixed the location.